.github/actions/setup/action.yml

```
@@ -0,0 +1,13 @@
+description: |
+  This action sets up the Bun environment and installs dependencies for the project.
+
+runs:
+  using: composite
+  steps:
+    - name: Setup bun
+      uses: oven-sh/setup-bun@v2
+
+    - name: Install dependencies
+      shell: bash
+      run: bun install --frozen-lockfile
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)